.github/workflows/push-event.yml

```diff
@@ -156,8 +156,48 @@ jobs:
           fi
 
       - name: Run Release Drafter
+        id: run-release-drafter
         uses: release-drafter/release-drafter@v6
         with:
           version: ${{ steps.android-version.outputs.VERSION_NAME }}
-          name: 'PSLab Version ${{ steps.android-version.outputs.VERSION_NAME }}'
-          tag: 'v${{ steps.android-version.outputs.VERSION_NAME }}'
+
+      - name: Delete Existing Assets
+        id: delete-assets
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const { data: assets } = await github.rest.repos.listReleaseAssets({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              release_id: ${{ steps.run-release-drafter.outputs.id }}
+            });
+
+             for (const asset of assets) {
+              await github.rest.repos.deleteReleaseAsset({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                asset_id: asset.id
+              });
+              console.log(`Deleted asset: ${asset.name}`);
+            }
+
+      - name: Upload APK Release Asset
+        uses: actions/upload-release-asset@v1
+        with:
+          upload_url: ${{ steps.run-release-drafter.outputs.upload_url }}
+          asset_path: app/build/outputs/apk/release/pslab-development-release.apk
+          asset_name: pslab-development-release.apk
+          asset_content_type: application/octet-stream
```

Comment:
Instead of using this action. Can you use github cli? Look at the gh manual, their is a gh release upload command which might be of use. It also has a --clobber flag to overwrite assets meaning you don't have to delete them

Reply:
@adityastic Done:)